# This is a simple path points message + yaw

float32[] x
float32[] y
float32 yaw
string dOne
string dTwo
